package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nuxtjs/auth",
-  "version": "4.9.0",
+  "version": "4.9.1",
```

Comment:
@tuzaobravado are there other parts of systems that use this library? And did we check if there are some breaking changes in this version?

Reply:
yes. the main project also uses this module.
both of them use the master branch as a version.
I don't think it's a breaking change as it's a new property check `disabledServerFetchUserOnce`